.circleci/config.yml

```diff
@@ -394,23 +372,13 @@ workflows:
           requires:
             - build
           main: true
-      - build_docker:
-          name: build_docker_data_dictionary
-          context: [hmpps-common-vars]
-          app_name: pic-data-dictionary
-          dockerfile_path: ./data_dictionary
```

Comment:
Is it worth removing the data_dictionary folder?

Reply:
yes - see https://dsdmoj.atlassian.net/browse/PIC-4392